sources/testme/js/kTestMe.js

```diff
@@ -270,7 +270,18 @@ var kTestMe = {
 		
 		this.classes[type].subClasses = subTypes;
 		for(var i = 0; i < subTypes.length; i++)
+		{
 			this.registerClass(subTypes[i]);
+			
+			if(subTypes[i].hasOwnProperty("isArray") && subTypes[i].isArray && subTypes[i].hasOwnProperty("arrayType") && subTypes[i].arrayType.hasOwnProperty("properties"))
```

Comment:
Regarding the check, if any of the attributes exist, then all of them exist, I would check for description and return from the method if it's missing.

You need to test with abstract attribute, such as advanced-search on filter or filter on response-profile.